tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -448,3 +482,76 @@ def test_numeric_decimal_incompatible(self):
 
         assert(conversion == reference)
 
+    #result: [('1E-28',), ('10000000000000000000000000.0',)]
+    def test_numeric_decimal_combined(self):
+        duckdb_conn = duckdb.connect()
+        decimals = pd.DataFrame(
+            data={
+                "0": [
+                    Decimal("0.0000000000000000000000000001"),
+                    Decimal("10000000000000000000000000.0")
+                ]
+            }
+        )
+        reference_query = """
+            CREATE TABLE tbl AS SELECT * FROM (
+                VALUES
+                (0.0000000000000000000000000001),
+                (10000000000000000000000000.0),
+            ) tbl(a);
+        """
+        duckdb_conn.execute(reference_query)
+        reference = duckdb.query("select * from tbl", connection=duckdb_conn).fetchall()
+        conversion = duckdb.query_df(decimals, "x", "select * from x").fetchall()
+        print(conversion)
```

Comment:
`assert` instead of `print` maybe? :)

Reply:
😅 I thought I had both, and print was there just to see what the result was

Reply:
```py
        reference_query = """
            CREATE TABLE tbl AS SELECT * FROM (
                VALUES
                    (1.234567890123456789012345678901234567890),
                    (123456789012345678901234567890123456789.0)
            ) tbl(a);
        """
 reference = duckdb.query("select * from tbl", connection=duckdb_conn).fetchall()
 print(reference)
```
----
```
[(1.2345678901234567,), (1.2345678901234568e+38,)]
```
That's not the object column conversion that's messing up

Reply:
I've fixed the issues, this time actually asserting ;)
It seems these issues all stem from not supporting conversion to hugeint backed decimals, so I added that and the problems magically disappeared :D